src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -2492,7 +2488,136 @@ def oil_water_surface_tension_model(p, t):
         )
 
     @staticmethod
-    def create_constant_ph_table(
+    def create_empty():
+        """
+        Creates a dummy (empty) PVT Table parameters with no valid data.
+        """
+        return PvtModelTableParametersDescription(
+            pressure_values=np.array([0.0]),
+            temperature_values=np.array([0.0]),
+            table_variables=[],
+            variable_names=[],
+            number_of_phases=-1,
+        )
+
+    def __eq__(self, other):
+        """
+        Need to re-implement the equality operator because ndarrays don't support equality, so the attr's generated
+        __eq__ function does not work.
+        """
+        if type(self) is not type(other):
+            return False
+
+        if len(self.table_variables) != len(other.table_variables):
+            return False
+
+        for array1, array2 in zip(self.table_variables, other.table_variables):
+            if not np.array_equal(array1, array2):
+                return False
+
+        return (
+            (self.pressure_values == other.pressure_values).all()
+            and (self.temperature_values == other.temperature_values).all()
+            and self.variable_names == other.variable_names
+            and self.pressure_std == other.pressure_std
+            and self.temperature_std == other.temperature_std
+            and self.number_of_phases == other.number_of_phases
+        )
+
+
+@attr.s(slots=True, eq=False)
+class PhPvtModelTableParametersDescription:
+    """
+    :ivar pressure_values:
+        Array like of sorted pressure values (m number of entries). [Pa]
+
+    :ivar enthalpy_values:
+        Array like of sorted enthalpy values (n number of entries). [J/Kg]
+
+    :ivar table_variables:
+        List of array like values for each property such as densities, specific heats,
+        enthalpies, etc.
+
+    :ivar variable_names:
+        List of property names
+
+    .. include:: /alfacase_definitions/list_of_unit_for_temperature.txt
+    .. include:: /alfacase_definitions/list_of_unit_for_density.txt
+    .. include:: /alfacase_definitions/list_of_unit_for_pressure.txt
+    .. include:: /alfacase_definitions/list_of_unit_for_standard_volume_per_standard_volume.txt
+    .. include:: /alfacase_definitions/list_of_unit_for_dimensionless.txt
+    """
+
+    pressure_values: Numpy1DArray = attr.ib(
+        validator=numpy_array_validator(dimension=1), repr=collapse_array_repr
+    )
+    enthalpy_values: Numpy1DArray = attr.ib(
+        validator=numpy_array_validator(dimension=1), repr=collapse_array_repr
+    )
+    table_variables: List[Numpy1DArray] = attr.ib(
+        validator=numpy_array_validator(dimension=1, is_list=True),
+        repr=collapse_array_repr,
+    )
+    variable_names: List[str] = attr.ib(validator=list_of_strings)
+
+    pressure_std = attrib_scalar(default=Scalar(1, "bar"), is_optional=True)
+    temperature_std = attrib_scalar(default=Scalar(15, "degC"), is_optional=True)
```

Comment:
Curiosity:
Since the PvtTable is type `PH`, should it have `enthalpy_std`?

(I am not familiar with PH tables, I am sorry for the dumb question)

Reply:
The `enthalpy_std` is calculated inside the `C++` `PhTable` class, but it's calculated from the standard temperature that comes from the table.

This calculation is one of that iterative calculations that we calculate the enthalpy given the temperature.